### PR TITLE
Fix button alignment for MultiMailViewer, MultiContactView, MultiSearchViewer (fixes #2443)

### LINKED ISSUE #2443: MailViewer/MultMailViewer/MultiSearchViewer have different button positions
Example gif:
anim

Notice how switching from MailViewer to MultiMailViewer shifts buttons horizontally and in search they are shifted vertically.

### PR DESCRIPTION
**Pull request by SnowyMouse**
This fixes #2443 as well as fixing the button positions for selecting multiple contacts.

after-small

### CODE REVIEW

File: src/contacts/MultiContactViewer.js [hunk 1]
```diff
@@ -27,11 +27,13 @@ export class MultiContactViewer {
 			return [
 				m(".fill-absolute.mt-xs.plr-l",
 					(contactView._contactList && contactView._contactList.list.getSelectedEntities().length > 0) ? [
-							m(".button-height"), // just for the margin
-							m(".flex-space-between", [
-								m(".flex.items-center", this._getContactSelectionMessage(contactView)),
-								m(actionBar)
-							])
+							// Add spacing so buttons for contacts also align with the regular client view's buttons
+							m(".header.pt-ml",
```
Comment (charlag) [inline src/contacts/MultiContactViewer.js:31]: We don't have a `header` class I think.
Contact viewer still jumps in multiselect for me if there is a name. I think we should make header at least the height of the name so it's the same size no matter if name is defined or not.

**Review by charlag — COMMENTED** [review]
Thanks! It works very well for mails but not for contacts yet

File: src/contacts/MultiContactViewer.js (hunk 1 above)
Comment (SnowyMouse) [reply src/contacts/MultiContactViewer.js:31]: I see!

For me, if I fill out both a name and a birthday (so there's two lines), it jumps for me, but not if I only have a name defined (only one line). I was testing with just a name filled out.

Comment (charlag) [reply src/contacts/MultiContactViewer.js:31]: Probably we can include it as-is because long mail subjects also move things around.

**Pull request merged**